### PR TITLE
Create special-characters

### PR DESCRIPTION
**Pull request by mattymatny**
Quick and dirty, feel free to adjust as needed.

### CODE REVIEW

File: _support-articles/windows/special-characters [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: article
+title: "Fixing crashes related to 'special' characters in Windows usernames"
+last_updated: 2019-01-28 13:00:00 +0000
+---
+
+Having an exclamation point (!), or some non-English Alphanumeric characters in your Windows Username (local windows user account) will cause the game to crash.
+
+In some cases you can fix this by changing the working directory the game uses by following the instructions [here](/support-articles/changing-working-direcotry/).
```
Comment (urielsalis) [inline _support-articles/windows/special-characters]: https://bugs.mojang.com/browse/MCL-9929 shows that this is not fixed by setting --workdir

File: _support-articles/windows/special-characters [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+If that does not resolve the issue, you will need to create a new local Windows username containing only English Alphanumernic characters.
+To be clear, this includes only (abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789) exactly as they are seen on this page.
+**NOTE:** Changing your Windows username will not correct the issue, you **NEED** to create a new local username.
+
+## Instructions for supported Windows Versions:
+
+Windows 7, 8, 8.1: https://support.microsoft.com/en-us/help/13951/windows-create-user-account#create-user-account=windows
```
Comment (urielsalis) [inline _support-articles/windows/special-characters]: Windows 
XP - http://support.microsoft.com/kb/279783
Vista - http://windows.	microsoft.com/en-us/windows/create-user-account#create-user-account=windows-vista
7 - http://windows.microsoft.com/en-us/windows/create-user-account#create-user-account=windows-7
8/8.1 - http://windows.microsoft.com/en-us/windows/create-user-account#create-user-account=windows-8
10 - https://support.microsoft.com/en-us/help/4026923/windows-10-create-a-local-user-or-administrator-account
Mac
10.6 Snow Leopard - http://support.apple.com/kb/ph6651
10.7 Lion - http://support.apple.com/kb/ph4600
10.8 Mountain Lion - http://support.apple.com/kb/PH11468

Comment (mattymatny) [reply _support-articles/windows/special-characters]: This one is windows only, xp and vista are unsupported, and the links for 7 8 8.1 are all to the same page

File: _support-articles/windows/special-characters (hunk 1 above)
Comment (mattymatny) [reply _support-articles/windows/special-characters]: Its not just for that issue. It covers all bases

File: _support-articles/windows/special-characters [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: article
+title: "Fixing crashes related to 'special' characters in Windows usernames"
+last_updated: 2019-01-28 13:00:00 +0000
+---
+
+Having an exclamation point (!), or some non-English Alphanumeric characters in your Windows Username (local windows user account) will cause the game to crash.
```
Comment (LordRalex) [inline _support-articles/windows/special-characters]: Username should not be cased here

File: _support-articles/windows/special-characters [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+In some cases you can fix this by changing the working directory the game uses by following the instructions [here](/support-articles/changing-working-direcotry/).
+
+If that does not resolve the issue, you will need to create a new local Windows username containing only English Alphanumernic characters.
+To be clear, this includes only (abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789) exactly as they are seen on this page.
+**NOTE:** Changing your Windows username will not correct the issue, you **NEED** to create a new local username.
+
+## Instructions for supported Windows Versions:
```
Comment (LordRalex) [inline _support-articles/windows/special-characters]: Casing here needs correcting. Supported should be upper.

File: _support-articles/windows/special-characters (hunk 2 above)
Comment (LordRalex) [reply _support-articles/windows/special-characters]: We have the room, so I'd rather just link properly for the versions that matter. No need for a note to save a line.

**Comment by LordRalex** [comment]
I'm not a fan of calling it a "Windows username". This term doesn't exist outside of this article, as they are called "Windows accounts". I'd rather we use the terms that MS uses, instead of mixing terms.

File: _support-articles/windows/special-characters (hunk 2 above)
Comment (mattymatny) [reply _support-articles/windows/special-characters]: Its the same link for each version, the different numbered links make no change to how the page is shown. The note would still be needed for people that don't know how to read through things.

**Comment by mattymatny** [comment]
> I'm not a fan of calling it a "Windows username". This term doesn't exist outside of this article, as they are called "Windows accounts". I'd rather we use the terms that MS uses, instead of mixing terms.

Changed all mentions to Windows User Account, or Windows User Account name, where needed.

File: _support-articles/windows/special-characters (hunk 2 above)
Comment (urielsalis) [reply _support-articles/windows/special-characters]: The # part in the URL makes the page jump to the correct place

**Review by urielsalis — APPROVED** [review]

**Comment by LordRalex** [comment]
Please validate your code renders correctly: https://deploy-preview-35--minecraftirc.netlify.com/support-articles/windows/special-characters/

Preview works on other pages, so I'm hesitant here.

**Comment by urielsalis** [comment]
sounds like he is doing a new line without 2 spaces in the previous line

File: _support-articles/windows/special-characters (hunk 2 above)
Comment (mattymatny) [reply _support-articles/windows/special-characters]: It has not done that once for me.

**Comment by mattymatny** [comment]
> Please validate your code renders correctly: https://deploy-preview-35--minecraftirc.netlify.com/support-articles/windows/special-characters/
> 
> Preview works on other pages, so I'm hesitant here.

It wasn't working cause I had not set it as a .md file yet, cause I never got an answer as to if that was to be used for this one or not.

File: _support-articles/windows/special-characters (hunk 2 above)
Comment (LordRalex) [reply _support-articles/windows/special-characters]: I'd have to agree with matt here, the URLs didn't redirect me properly.

**Pull request merged**